src/studio/src/designer/frontend/shared/components/FileSelector.tsx

```diff
@@ -13,32 +12,6 @@ export interface IFileSelectorProps extends StyledComponentProps {
   accept?: string;
 }
 
-const useStyles = makeStyles({
```

Comment:
Won't this remove all focus/hover states and such as well?

Reply:
No, those stylings are implied somewhere else. The code I removed here didn't originally belong to TopToolbarButton at all.